Gems/ROS2/Code/Source/ROS2SystemComponent.cpp

```diff
@@ -148,6 +148,11 @@ namespace ROS2
         }
     }
 
+    float ROS2SystemComponent::GetExpectedSimulationLoopTime() const
+    {
+        return aznumeric_cast<float>(m_simulationClock.GetMedianExecutionTime()) / 1e6;
+    }
```

Comment:
Would jt be better to return an [AZStd::chrono::seconds](https://en.cppreference.com/w/cpp/chrono/duration)?